### PR TITLE
UI enhancement of Chat Screen

### LINKED ISSUE #11: More effective UI of chat bubble
Need to Modify the Chat Bubble align the time and message section
and the buttom TextField

WhatsApp Image 2023-10-20 at 17 00 41_c68a80fa

### PR DESCRIPTION
**Pull request by JayrajSinh16**
@RitwikSingh28
Changed the shape of the chat Bubble and the Style of the Message text field 

WhatsApp Image 2023-10-22 at 00 35 24_6e1b1592

### CODE REVIEW

File: lib/shared/widgets/gradient_background.dart [hunk 1]
```diff
@@ -37,7 +37,7 @@ class UniDirectionalBackground extends StatelessWidget {
           gradient: LinearGradient(
         colors: [
           Color(0xFF6CAFB4),
-          Colors.white,
+          Color.fromARGB(255, 223, 242, 187),
```
Comment (RitwikSingh28) [inline lib/shared/widgets/gradient_background.dart]: @JayrajSinh16 

**Don't deviate from the pre-determined colorscheme**. If you wish to make changes that are outside the scope of your issue, please discuss it first if what you wish to change conforms to the general flow of the project or not.

**Please revert all the changes made in the `LinearGradient` widget**

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
@JayrajSinh16 
There are some problems with the overall look, which I have described in the above section.

Moreover, here's another problem:
* The timestamp font-size seems to be bigger than necessary
* The `borderRadius` is too rounded

File: lib/shared/widgets/gradient_background.dart (hunk 1 above)
Comment (JayrajSinh16) [reply lib/shared/widgets/gradient_background.dart]: @RitwikSingh28 
yes sir I have revert the changes made in the Linear Gradient widget.
Rounded Border issue resolved.
time-stamp font size reduced.
Sir please review the same.

WhatsApp Image 2023-10-22 at 09 34 51_a5eed267

**Review by RitwikSingh28 — APPROVED** [review]
Good work @JayrajSinh16 !
